sysconfig/cloudinit_test.go

```diff
@@ -360,37 +360,38 @@ func (s *sysconfigSuite) TestRestrictCloudInit(c *C) {
 	}
 
 	for _, t := range tt {
+		comment := Commentf(t.comment)
```

Comment:
Technically this should be:
```suggestion
		comment := Commentf("%s", t.comment)
```

Because in theory `t.comment` may contain formatting expressions.